package.json

```diff
@@ -149,7 +149,7 @@
   },
   "dependencies": {
     "@gnosis.pm/safe-contracts": "1.1.1-dev.2",
-    "@gnosis.pm/safe-react-components": "^0.1.3",
+    "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components#8b0de7868b69e520249d724816f0c17ea19e2a47",
```

Comment:
is this still actual? no new version released?

Reply:
no new version yet

Reply:
I think now, you can use the last release @fernandomg